File: lib/api/model/Patient.dart
class Patient {
  int id;
  String nom;
  DateTime date;
  double score;
  bool sick;

  Patient({this.id, this.nom, this.date, this.score, this.sick});

  static Patient fromJson(Map<String, dynamic> json) {
    return Patient(
        id: json["id"] as int,
        nom: json["nom"] as String,
        date: json['date'] != null
            ? DateTime.parse(json['date'] as String)
            : null,
        score: json["score"] as double,
        sick: json["sick"] as bool);
  }
}

File: src/main/java/com/tpjpa/demo/moduls/Patient.java
package com.tpjpa.demo.moduls;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;
import org.springframework.format.annotation.DateTimeFormat;

import javax.persistence.*;
import javax.validation.constraints.Size;
import java.util.Date;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
@Table(name="Patients")
public
class Patient {
    @Id
    @GeneratedValue(strategy = GenerationType.SEQUENCE)
    private Long id;
    private String nom;
    @Temporal(TemporalType.DATE)
    @DateTimeFormat(pattern = "yyyy-MM-dd")
    private Date birth;
    private double score;
    private boolean isSick;

}